src/Otl.php

```diff
@@ -1539,6 +1539,9 @@ function _applyGSUBrulesIndic($usetags, $scriptTag, $langsys, $is_old_spec)
 									continue;
 								}
 
+								if ($ptr + 1 == count($this->OTLdata)) {
```

Comment:
Use strict comparison here, no reason not to.

Reply:
> Use strict comparison here, no reason not to.

I have modified that.